spec/features/user_edits_entry_spec.rb

```diff
@@ -0,0 +1,26 @@
+feature "User edits entry" do
+  scenario "and the entry body is updated" do
+    user = create(:user)
+    create(:entry, user: user, body: "Original body")
+
+    login_as(user)
+    visit entries_path
+    click_link "Edit this entry"
+    fill_in :entry_body, with: "New body"
+    click_button "Save Entry"
+
+    expect(page).to_not have_content("Original body")
+    expect(page).to have_content("New body")
+  end
+
+  scenario "and the entry belongs to another user" do
```

Comment:
WDYT about making this a controller spec instead?

I think it'll be a bit shorter, and a bit faster.